universe/wrappers/vectorize.py

```diff
@@ -58,6 +56,7 @@ def _seed(self, seed):
 
 class WeakUnvectorize(Unvectorize):
     def __init__(self, env, i):
+        # What is i? Seems not to be used anywhere?
```

Comment:
Anyone want to chime in on this one? There are no obvious usages of this attribute, and removing it doesn't break any tests. Artefact of some earlier capability maybe?